Specs/Sequence/Chinese/URLModel.json

```diff
@@ -0,0 +1,527 @@
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "https://luis.ai?action=add",
+        "Start": 13,
+        "End": 38,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "https://luis.ai?action=add"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Please visit https://luis.xxx?action=add for more information",
+    "NotSupportedByDesign": "javascript, python",
+    "Results": []
+  },
+  {
+    "Input": "working..is isn't a valid domain name.",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "7.am is more likely a datetime string rather than a valid domain name.",
+    "NotSupportedByDesign": "python",
+    "Results": []
+  },
+  {
+    "Input": "Please visit https://7.am for more information.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "https://7.am",
+        "Start": 13,
+        "End": 24,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "https://7.am"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "Please visit 27.pm and s7.am for more information.",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "27.pm",
+        "Start": 13,
+        "End": 17,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "27.pm"
+        }
+      },
+      {
+        "Text": "s7.am",
+        "Start": 23,
+        "End": 27,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "s7.am"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "网址是microsoft.com",
+    "Notsupport": "java, javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "microsoft.com",
+        "Start": 3,
+        "End": 15,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "microsoft.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "微软的官网是microsoft.com哦",
+    "Notsupport": "java, javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "microsoft.com",
+        "Start": 6,
+        "End": 18,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "microsoft.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "微软的官网是：microsoft.com",
+    "Notsupport": "java, javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "microsoft.com",
+        "Start": 7,
+        "End": 19,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "microsoft.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "微软的官网是：www.microsoft.com",
+    "Notsupport": "java, javascript",
+    "NotSupportedByDesign": "python",
+    "Results": [
+      {
+        "Text": "www.microsoft.com",
+        "Start": 7,
+        "End": 23,
+        "TypeName": "url",
+        "Resolution": {
+          "value": "www.microsoft.com"
+        }
+      }
+    ]
+  },
+  {
+    "Input": "はmicrosoft.comです",
```

Comment:
Chinese/URLModel.json should only contain cases that we want to validate for Chinese.
Can you move this Japanese case to it's own spec and make sure the tests run for it?